addons/rose/addon/components/rose/list/value/item/index.hbs

```diff
@@ -0,0 +1,8 @@
+<li class="rose-list-value-item" id={{concat "item-value-" this.id}}>
```

Comment:
Struggled a bit here to find an aria label to add. Ended up with using an unique id coz there isn't a label within this component, unless a `title` or `label` can be passed in. Thoughts?